common/middleware/sentry-enrich-scope.js

```diff
@@ -21,6 +21,8 @@ module.exports = function sentryEnrichScope(req, res, next) {
   if (user) {
     Sentry.setUser({
       id: user.userId,
+      username: user.username,
```

Comment:
Need to double check if we can add this information.

Reply:
@martin-ballhatchet-moj what are your thoughts on us adding this to our issue logging? Does it count as PPI that may cause any GDPR issues?

cc/ @jason-sims

Reply:
Hi Dom. This does count as Personal information, as usernames are normally a permutation of a real user's name. As always, we do need to be careful when we expose this kind of data. However, in this case, we are publishing to an official tool (Sentry), which relies on good authentication. We are also only publishing in the case of a genuine error in order to troubleshoot. I am fine with adding it for now - and then we can review later whether to remove it again.